x-pack/apm-server/sampling/eventstorage/storage_manager.go

```diff
@@ -46,6 +47,12 @@ const (
 
 	// diskUsageFetchInterval is how often disk usage is fetched which is equivalent to how long disk usage is cached.
 	diskUsageFetchInterval = 1 * time.Second
+
+	// dbStorageLimitFallback is the default fallback storage limit in bytes
+	// that applies when disk threshold cannot be enforced due to an error.
+	dbStorageLimitFallback = 5 << 30
```

Comment:
How did we come up with 5GB? Along the same lines, what was the reasoning for choosing 3GB default previously? Should we keep the fallback to 3GB? - If the previous value was also chosen randomly then no objection with 5GB.

Reply:
I wonder if we should keep falling back to `3GB`? This would yield in the same result for upgraded instances where the newly introduced storage limit cannot be fetched.

Reply:
No problem with keeping it at 3GB for fallback